From ad1f47c3f2391914028d9ced03e68ba2da95bb9e Mon Sep 17 00:00:00 2001
From: Sven Joachim <svenjoac@gmx.de>
Date: Wed, 25 Nov 2015 19:07:57 +0100
Subject: [PATCH 2/2] Bump bash-completion build dependency to 1:2.1-4.2~

Building the package with an older bash-completion version would
reintroduce the conffile which is highly undesirable.
---
 debian/changelog | 1 +
 debian/control   | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/debian/changelog b/debian/changelog
index f34753d..5654b91 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -2,6 +2,7 @@ apt-xapian-index (0.47+nmu2) unstable; urgency=medium
 
   * Non-maintainer upload.
   * Remove /etc/bash_completion.d/axi-cache on upgrades. Closes: #806234
+  * Bump bash-completion build dependency to 1:2.1-4.2~ (see #668254).
 
  -- Sven Joachim <svenjoac@gmx.de>  Wed, 25 Nov 2015 19:07:27 +0100
 
diff --git a/debian/control b/debian/control
index d471c3f..341345c 100644
--- a/debian/control
+++ b/debian/control
@@ -5,7 +5,7 @@ Maintainer: Enrico Zini <enrico@debian.org>
 Uploaders: David Paleino <dapal@debian.org>
 Build-Depends: debhelper (>= 7.0.50~)
 Build-Depends-Indep: help2man, python (>= 2.6.6-3~), dh-python,
- bash-completion (>= 1:1.0-1~), 
+ bash-completion (>= 1:2.1-4.2~), 
  python-xapian (>= 1.0.2),
  python-apt (>= 0.7.93.2),
  python-debian (>= 0.1.14),
-- 
2.6.2

